httpServer.js

```diff
@@ -1,5 +1,29 @@
 const http = require('http');
 const app = require('./app');
+var mysql = require('mysql');
+
+var connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'root123!',
+    database: 'appDB'
+});
+
+//connection.connect();
+connection.connect(function (error) {
+    //callback
+    if (!!error) {
+        console.log('Error');
+    }
+    else {
+        console.log('Connected');
+    }
+});
+
+connection.query('SELECT * FROM CATEGORIES', function (err, rows, fields) {
```

Comment:
what actually are you doing with this query?

Reply:
I would export this in a module that does just the connections and exports it. making available through all app.